bin/reth/src/node/mod.rs

```diff
@@ -444,24 +475,43 @@ impl Command {
         }
 
         let factory = reth_executor::Factory::new(self.chain.clone());
+
+        let default_stages = if continuous {
+            let continuous_headers =
+                HeaderStage::new(header_downloader, consensus.clone()).continuous();
+            let online_builder = OnlineStages::builder_with_headers(
+                continuous_headers,
+                consensus.clone(),
+                body_downloader,
+            );
+            DefaultStages::<H, B, U, reth_executor::Factory>::add_offline_stages(
+                online_builder,
+                updater.clone(),
+                factory.clone(),
+            )
+        } else {
+            DefaultStages::new(
+                consensus.clone(),
+                header_downloader,
+                body_downloader,
+                updater.clone(),
+                factory.clone(),
+            )
+            .builder()
```

Comment:
we weren't calling `builder()` before, were we?

Reply:
we didn't need to before because `DefaultStages` implements `StageSet`, and the builder conversion happens internally in `set`